platform/ui/src/components/Modal/Modal.jsx

```diff
@@ -24,13 +27,23 @@ const Modal = ({
   onClose,
   children,
 }) => {
+  const { hide } = useModal();
+
+  const handleClose = () => {
+    hide();
+  };
+
   const renderHeader = () => {
     return (
       title && (
-        <header>
-          <h4>{title}</h4>
+        <header className="mb-6 pb-4 border-b border-secondary-main">
+          <Typography variant="h4">{title}</Typography>
           {closeButton && (
-            <button data-cy="close-button" onClick={onClose}>
+            <button
+              className="absolute top-0 right-0 bg-primary-main focus:outline-none text-white rounded-full text-2xl w-8 h-8 flex justify-center items-center -mr-3 -mt-3"
+              data-cy="close-button"
+              onClick={onClose}
+            >
               ×
```

Comment:
Do we have an icon for this X? I can see that it's a little off centred here when I open the modal, maybe switching to an icon would help.

Reply:
we have an icon for "cancel", but I've just created a new close icon based on the close icon to match better and replaced for the new icon. Here's how it displays: 

![image](https://user-images.githubusercontent.com/1905961/85810700-55907600-b732-11ea-88e9-84567f1f38fd.png)